### DISCUSSION TITLE
Pilot + instructor mode, with ability to take over control

### DISCUSSION THREAD

**Discussion by piotr-piatkowski** [ExpressLRS/ExpressLRS#2817] [Ideas]
I sometimes let newbie pilots try flying with my quads - but even after they train on the simulator I'm a bit afraid, and usually keep my finger on the arm button to disarm quickly in case of any problems.

It would be very handy to be able to connect two transmitters simultaneously to the same quad - one for the newbie pilot and second for the instructor. Instructor could take the control at any moment by using a switch. Then - instructor can fly away to some safe location (like - away from any obstacles and say 30 feet AGL) and let newbie pilot fly - but at any moment he can take control back.

What do you think? Would it be handy? Would it be technically possible?

**Comment by jurgelenas** [comment]
It's already possible with EdgeTX trainer setup: https://manual.edgetx.org/bw-radios/radio-settings/trainer

**Reply by piotr-piatkowski** [reply]
Oh, great - thanks. I wasn't aware of that, I'll check this out.

**Reply by piotr-piatkowski** [reply]
OK, I see this is possible with using internal RF module - but is it possible with only two ELRS modules? Maybe using BLE or WiFi connection?
